File: Getter and Setter Methods.cpp
#include <iostream>
using namespace std;

class Person {
private:
    string name;  // Private attribute

public:
    Person(string n) : name(n) {}

    string getName() {
        return name;
    }

    void setName(string n) {
        name = n;
    }
};

int main() {
    Person person("John");
    cout << person.getName() << endl;
    person.setName("Alice");
    cout << person.getName() << endl;
    return 0;
}

File: Getter and Setter Methods.py
# Getter and Setter methods allow access and modification of private class variables.
class Person:
    def __init__(self, name):
        self.__name = name  # Private attribute

    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

person = Person("John")
print(person.get_name())
person.set_name("Alice")
print(person.get_name())
